sanic/response.py

```diff
@@ -79,7 +79,9 @@ def __init__(self, body=None, status=200, headers=None,
         self.content_type = content_type
 
         if body is not None:
-            self.body = body.encode('utf-8')
+            self.body = body
+            if type(body) is str:
```

Comment:
isinstance was not used because in many test performance seem slower than type(...)